extensions/target-specific/chromium/blink/fragment-tree/fragment-tree.js

```diff
@@ -43,13 +59,8 @@ Loader.OnLoad(function() {
                             "You may still specify a blink::NGPhysicalFragment explicitly.";

```

Comment:
We should also update this error message to say "specify a blink::NGPhysicalFragment **or blink::LayoutObject** explicitly".

Reply:
Done